api/views/phones.py

```diff
@@ -462,12 +469,16 @@ def inbound_sms(request):
     relay_number.remaining_texts -= 1
     relay_number.texts_forwarded += 1
     relay_number.save()
-    return response.Response(status=201, data={"message": "Relayed message to user."})
+    return response.Response(
+        {"message": "Relayed message to user."},
```

Comment:
Is the `message` needed, so that _something_ is returned?

Reply:
Not needed. Removed.